src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -922,9 +924,12 @@
      * Specifies the text to be placed at the bottom of each output file.<br/>
      * If you want to use html, you have to put it in a CDATA section, <br/>
      * e.g. <code>&lt;![CDATA[Copyright 2005, &lt;a href="http://www.mycompany.com">MyCompany, Inc.&lt;a>]]&gt;</code>
-     * <br/>
+     * <br>
+     * <strong>Note:<strong>If the project has the property <code>project.build.outputTimestamp</code>, its year will
+     * be used as {currentYear}. This way it is possible to generate reproducible javadoc jars.
+     * <br>
      * See <a href="https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#bottom">bottom</a>.
-     * <br/>
+     * <br>
```

Comment:
If this is XHTML, then `<br/>` is the correct way to write a standalone (unpaired) tag that is already closed. Same for the other two `<br>` above.
In case of doubt, pass the rendered output to http://validator.w3.org/

Reply:
Not really, HTML5. Javadoc will complain.

Reply:
It depends on the Java version. As the minimum is Java 8, it should be be HTML (I think v4)